cmake/XRootDDefaults.cmake

```diff
@@ -18,7 +18,7 @@ option( ENABLE_XRDCL     "Enable XRootD client."
 option( ENABLE_TESTS     "Enable unit tests."                                             FALSE )
 option( ENABLE_HTTP      "Enable HTTP component."                                         TRUE )
 option( ENABLE_PYTHON    "Enable python bindings."                                        TRUE )
-option( PIP_VERBOSE      "Turn on --verbose flag for pip during Python bindings install." FALSE )
+option( PIP_OPTIONS      "pip options to use during the Python bindings install."         STRING )
```

Comment:
@henryiii Sorry to bother you with a CMake question, but can you remind me how to properly escape an option like there where there will be spaces in the string? I assume that there is a standard way of doing things like this that you use in the Scikit-build world.

Reply:
Nevermind this is a non-issue. No special escaping needs to be done.